util/coding.h

```diff
@@ -354,6 +354,13 @@ inline Slice GetSliceUntil(Slice* slice, char delimiter) {
 }
 
 template<class T>
+#ifdef ROCKSDB_UBSAN_RUN
+#if defined(__clang__)
+__attribute__((no_sanitize("alignment")))
```

Comment:
I can change it - I don't really understand the difference, all the documentation I could find mentioned `no_sanitize`, not `__no_sanitize__` (https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html#disabling-instrumentation-with-attribute-no-sanitize-undefined).

Anyway, changed.